### PR TITLE
NDSR-907: Migrate accounts CSV into TOML.

### PR DESCRIPTION
**Pull request by mpapierski**
Ref: https://casperlabs.atlassian.net/browse/NDRS-907

This PR replaces accounts.csv into a structured TOML file in preparation to hold delegators data there as well.

New structure:

```toml
[[accounts]]
public_key = "01522ef6c89038019cb7af05c340623804392dd2bb1f4dab5e4a9c3ab752fc0179"
balance = "1000000000000000000000000000"
bonded_amount = "0"
```

### CODE REVIEW

**Review by EdHastingsCasperLabs — APPROVED** [review]

**Review by sacherjj — APPROVED** [review]

**Review by henrytill — APPROVED** [review]
Looking good

File: utils/casper-tool/casper-tool.py [hunk 1]
```diff
@@ -290,13 +290,22 @@ def create_node(
...
+
+    accounts = []
     for id, key_hex in items:
         motes = 1000000000000000
         weight = 10000000000000
-        output_file.write("{},{},{}\n".format(key_hex, motes, weight))
+        account = {
+            'public_key': key_hex,
+            'balance': f'{motes}',
+            'staked_amount': f'{weight}'
```
Comment (goral09) [inline utils/casper-tool/casper-tool.py:304]: Is it "bonded amount" or "staked amount"? You need to be consistent.

**Pull request merged**